documentation/src/main/asciidoc/topics/proc_generating_protostream_sci.adoc

```diff
@@ -1,20 +1,7 @@
 [id='generating_protostream_sci-{context}']
-= Generating Serialization Context Initializers
-A ProtoStream `SerializationContext` contains Protobuf type definitions for
-custom Java objects, loaded from Protobuf schemas, and the accompanying
-marshallers for those objects.
-
-{brandname} provides a `protostream-processor` artifact that processes Java
-annotations in your classes at compile time. The processor generates Protobuf
-schemas, marshallers, and a concrete implementation of the
-`SerializationContextInitializer` interface that you can use to initialize a
-ProtoStream `SerializationContext`.
-
-[NOTE]
-====
-By default, implementation names are the annotated class name with an "Impl"
-suffix.
-====
+= Marshalling Custom Java Objects
+To marshall custom Java objects, you first annotate those objects and then generate an implementation of the ProtoStream `SerializationContextInitializer` interface that describes your Java objects.
```

Comment:
Instead of "describes your Java objects" I think we should be saying that you specify the objects to include in the SerializationContextInitializer. However, I'm not sure how best to word that.